packages/utils/src/money/format-balance/format-balance.ts

```diff
@@ -0,0 +1,22 @@
+import { abbreviateNumber } from '../../abbreviate-number/abbreviate-number';
+import { removeCommas } from './remove-commas';
+
+/**
```

Comment:
When I dug into this it seems like we could perhaps refactor this a bit based on the data we get from the API.